contribs/gmf/apps/mobile/index.html

```diff
@@ -62,6 +62,12 @@
       <gmf-disclaimer
         gmf-disclaimer-map="::mainCtrl.map">
       </gmf-disclaimer>
+      <div class="visible-sm">
+        <div class="alert alert-info alert-dismissible fade in" role="alert">
+          <button type="button" class="close" data-dismiss="alert" aria-label="{{'Close' | translate }}"><span aria-hidden="true" class="fa fa-times"></span></button>
+          <span>{{"You're using the mobile version. Check out the" | translate }} <a href="http://camptocamp.github.io/ngeo/master/examples/contribs/gmf/apps/desktop/">{{'standard version' | translate}}</a>.</span>
```

Comment:
@rbovard you speak about the URL?
If yes it will be rewrite with a pyramid route for the real application :-)